platforms/avr/clockless_trinket.h

```diff
@@ -323,7 +323,11 @@ class ClocklessController : public CPixelLEDController<RGB_ORDER> {
 #define DUSE (0xFF - (DADVANCE-1))
 
 // Silence compiler warnings about switch/case that is explicitly intended to fall through.
-#define FL_FALLTHROUGH __attribute__ ((fallthrough));
+#if defined(__GNUC__) && __GNUC__ >= 7
+ #define FL_FALLTHROUGH __attribute__ ((fallthrough));
+#else
+ #define FL_FALLTHROUGH ((void)0);
```

Comment:
you could just
#define FL_FALLTHROUGH
here, the ((void)0) trick is to avoid adding no instruction before the semicolon (in case it's not in the macro) so here it can be defined to 'empty' and works

Reply:
I like the comment trick I will try it.